components/menu/test/menu-rtl.visual-diff.js

```diff
@@ -38,7 +38,7 @@ describe('d2l-menu rtl', () => {
 			// this scenario also tests height change when going from 1 menu item to 2 within nested menu
 			await page.$eval('#nested-item', item => {
 				return new Promise((resolve) => {
-					item.addEventListener('d2l-hierarchical-view-show-complete', resolve, { once: true });
+					item.addEventListener('d2l-hierarchical-view-show-complete', () => requestAnimationFrame(resolve), { once: true });
```

Comment:
Now that the `resize` event is a frame later, we need to wait a frame for the visual-diff tests.  We could also listen for the `d2l-hierarchical-view-resize` event here instead, but I like testing that the resize is only one frame after the `show` event.  This doesn't change the order of events - the `show` event always came before the `resize` one in the `reduced-motion` case.